app/gameengine.py

```diff
@@ -5156,7 +5297,8 @@ def handle_effect_resolution_order_cards(self, player_id:str, action_data:dict):
         bottom = self.current_decision["bottom"]
 
         # The cards are in the order they should be put at that location.
-        for card_id in card_ids:
+        # The ids needs to be reversed if they go on top
+        for card_id in card_ids[::1 if bottom else -1]:
```

Comment:
this line makes me think too hard about if :: is an operator or something and if it's ::(1 or -1) or or (::1) or (-1). Haha...
I assume it has to be the one that actually works.
Is it too annoying if I ask to make it:
if not bottom:
   card_ids = card_ids[::-1]